jobs/integr8ly/tower-clean-uninstall.yaml

```diff
@@ -0,0 +1,113 @@
+---
+
+- job:
+    name: tower-clean-uninstall
+    description: 'Uninstalls Integreatly from fresh pure cluster created by Ansible Tower tooling where Integreatly has never been installed.'
+    project-type: pipeline
+    sandbox: false
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 20
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://github.com/integr8ly/installation.git'
+          description: 'Repository of the Integreatly installer'
+      - string:
+          name: BRANCH
+          default: 'master'
+          description: 'Branch of the installer repository'
+      - string:
+          name: RECIPIENTS
+          default: integreatly-qe@redhat.com
+          description: 'Whitespace- or comma-separated list of recipient addresses'
+    triggers:
+      - timed: 'H H(0-3) * * *'
+    dsl: |
+
+        def setClusterAdminCredentials() {
```

Comment:
There is inconsistent tab size for this method (2 spaces vs 4 spaces elsewhere).

Reply:
oh, yes, I copy-pasted this piece. Will fix the indentation.

Reply:
@pawelpaszki fixed, I hope.

Reply:
I think lines #33 and #34 should be indented by 2 more spaces as well

Reply:
grrr, good catch!

Reply:
ehm, fixed.